src/shogun/classifier/Perceptron.cpp

```diff
@@ -73,8 +73,9 @@ void CPerceptron::iteration()
 		{
 			converged = false;
 			const auto gradient = learn_rate * true_label;
-			bias += gradient;
+			put("bias", bias+gradient);
```

Comment:
This is the updated version of perceptron with the new `put()`.

Reply:
looks good!